lib/elixir/test/elixir/calendar/holocene.exs

```diff
@@ -24,12 +24,12 @@ defmodule Calendar.Holocene do
 
   @impl true
   def date_to_string(year, month, day) do
-    "#{year}-#{month}-#{day} (HE)"
+    "#{year}-#{pad(month)}-#{pad(day)} #{inspect(__MODULE__)}"
```

Comment:
Here and in other places you can use the following. Even though Date.new etc doesn't support year > 9999, the following function happens to:

```suggestion
    Calendar.ISO.date_to_string(year, month, day) <> " " <> inspect(__MODULE__)
```